contributing-to-open-source.md

```diff
@@ -0,0 +1,249 @@
+the project.
+
+But first, see if anyone else have already stumbled on the same
+problem:
+
+- Search among issues to see if it is already reported.
+  - Remember to search among closed issues too, since the default search
+    view on Github only searches among open issues.
+  - Searching for error messages, exception texts or stacktraces can
+    help locating issues for the same problem.
+
+If this is a closed issue, you may find workarounds or find that it is
+fixed in existing or upcoming releases.  If you find an open issue for
+the problem you can subscribe to it to track developments.
+
+If you don't find any issue, see if upgrading to the latest patch or
+minor version helps, or even the major version, if you can.  There are
+often nightly builds of the main development branch that you can try
+to test too.
+
+
+## Submitting issues
+
+If it is an unreported problem, do submit an issue!  Projects often
+have a template that they want you to fill in, but if not make sure to
+include information that helps the maintainers reproduce and
+understand the problem, e.g.:
+
+- Version that the problem was found in
+- Language/framework versions
+- OS versions
+- Any error messages or stack traces
+  - Include them as text in code blocks, not screen shots. Text is
+    searchable for others who have the same problem.
+- How to reproduce the problem
+  - E.g. a code example or command line.
+  - Even better is a standalone minimal code project that reproduces the bug.
+- Any workarounds you may have found
+
+If you later find out that the problem was not with the project itself
+but how you used it, close the issue with a comment explaining what
+you did to solve it.  This will help people who have the same problem
+later when they find the issue.
+
+
+## Improving issues
+
+If you find an open issue for a problem you've run into, see if you
+can flesh out that issue by providing more information that can help
+solve it.  If the issue already contains everything you know, you can
+add your vote to how important it is by adding a `+1` emoji by
+clicking the face button on the original issue report.  (Don't add
+spammy "me too!" comments.)
+
+Adding this to the issue also have the bonus that you get subscribed
+to it automatically and can track any further discussions or fixes.
+
+
+## Following up on issues
+
+If you report an issue and the developers respond, make sure you
+follow up and help them with any further information that you can
+provide.  Issues where there's no responses to questions from the
+developers tend to be marked as stale and closed after some time.
+
+When a bug fix is proposed in a pull request have a look at it, if
+possible test it, and provide feedback.
+
+
+# Bug fixes
+
+Think you can fix a bug you've found in an open source project?  Start
+by reporting it as described above, and add a note that you will try
+to fix it yourself.
+
+Fixing bugs in open source is not much different to doing in your
```

Comment:
Maybe "...not much different than in your normal projects, but...".
Not sure about the use of normal, but couldn't think of anything better myself.